scenarios/15a_The_Preparations.cfg

```diff
@@ -161,10 +161,15 @@
         {MSGm_Gawen _"I understand. My fifteen best soldiers. When should I go?"}
 #endif
 #ifdef HARD
-        #po: HARD difficulty, somewhat vaguer than the other two:
+        #po: HARD difficulty, somewhat vaguer than the previous two:
         {MSG_Gwidle _"...so, you'll take twelve of your troops and search for that mage, to try to convince him to join our cause."}
         {MSGm_Gawen _"I understand. Twelve soldiers. When should I go?"}
 #endif
+#ifdef NIGHTMARE
```

Comment:
Well 9 troops is really kind of brutal but let's go with it for now.

Reply:
it *is* "`NIGHTMARE`" difficulty, after all!